app-frontend/src/app/pages/projects/edit/exports/new/new.html

```diff
@@ -35,6 +36,26 @@ <h5 class="sidebar-title">Export</h5>
       </button>
     </div>
     <i class="icon-info"></i>
+  </li> -->
+  <!-- Dropdown for output processing -->
+  <li class="separator">
+    <span class="label">Output Processing</span>
+    <div class="dropdown btn-group fixedwidth" uib-dropdown uib-dropdown-toggle>
```

Comment:
this doesn't seem to update when I select a raw

Reply:
It should be fixed in the latest commit.